plugins/logger/frontend/public/javascripts/countly.views.js

```diff
@@ -369,6 +370,28 @@ window.LoggerView = countlyView.extend({
             currentLogItem = currentLogItem[0];
             $("#logger-selector").clySelectSetSelection(currentLogItem.value, currentLogItem.name);
         }
+    },
+    renderPageWarningAlert: function(message) {
+        $(".routename-logger #logger-alert").html("");
+        var warningAlertContainer = '<div class="event-remind-tooltip" style="display: block;">' +
+                    '<div class="content">' +
+                            '<span class="prefix"></span>' +
+                            '<span class="remind-context">' + message + '</span>' +
+                            '<i class="fas fa-times"></i>' +
+                    '</div>' +
+                '</div>';
+        setTimeout(function() {
+            $(".routename-logger #logger-alert").append(warningAlertContainer);
+            $(".event-remind-tooltip .fa-times").off("click").on("click", function(e) {
+                $(e.currentTarget.parentElement.parentElement).remove();
+            });
+        }, 0);
+    },
+    displayLoggerStateWarningAlertIfNecessary: function() {
+        var activeAppPlugins = countlyGlobal.apps[countlyCommon.ACTIVE_APP_ID].plugins;
+        if (activeAppPlugins && activeAppPlugins.logger.state === 'off') {
```

Comment:
I think there could still be a state when plugins object is defined (because the app has configs modified previously), but the logger object is not, because it does not propagate automatically to all apps, I think
so logger.state check would throw an exception

Reply:
good to know as I did not expect that would be the case if the plugin is loaded. I just put in the null pointer check for logger plugin.